src/app/public/modules/split-view/split-view-adapter.service.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  ElementRef,
+  Injectable,
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+import {
+  MutationObserverService,
+  SkyAppWindowRef
+} from '@skyux/core';
+
+import {
+  fromEvent,
+  Subject
+} from 'rxjs';
+
+import {
+  take,
+  takeUntil
+} from 'rxjs/operators';
+
+@Injectable()
```

Comment:
This will need an `@internal` decorator for jsDocs.

Reply:
Done